### ISSUE TITLE
bug: Integrate Existing Consolidated CloudTrail - terraform rerun deletes manually attached SNS topic

### ISSUE THREAD

**Issue by dmcnaught** [lacework/terraform-aws-cloudtrail#110]
**Describe the bug**
  We have reconfigured our Cloudtrail integration to use terraform since we have also added two new AWS accounts and our old integration was done manually.
I followed instructions [here](https://docs.lacework.com/onboarding/aws-cloudtrail-integration-with-terraform#scenario-5---integrate-existing-consolidated-cloudtrail-and-configuration-assessment): 
There is this section:
> If you do not have an existing SNS topic configured on the existing CloudTrail, the Terraform module automatically creates one, but you must manually attach the SNS topic to the existing CloudTrail.

After doing the manual attachment, if I run the terraform apply again it will remove the manual attachment:
```hcl
Terraform will perform the following actions:

  # module.altoira_cloudtrail[0].aws_sns_topic_policy.default[0] will be updated in-place
  ~ resource "aws_sns_topic_policy" "default" {
        id     = "arn:aws:sns:us-east-1:<redacted>:lacework-ct-sns-de98b010"
      ~ policy = jsonencode(
          ~ {
              ~ Statement = [
                    {
                        Action    = "SNS:Publish"
                        Effect    = "Allow"
                        Principal = {
                            Service = "cloudtrail.amazonaws.com"
                        }
                        Resource  = "arn:aws:sns:us-east-1:<redacted>:lacework-ct-sns-de98b010"
                        Sid       = "AWSCloudTrailSNSPolicy20131101"
                    },
                  - {
                      - Action    = "SNS:Publish"
                      - Condition = {
                          - StringEquals = {
                              - "AWS:SourceArn" = "arn:aws:cloudtrail:us-east-1:<redacted>:trail/management-events"
                            }
                        }
                      - Effect    = "Allow"
                      - Principal = {
                          - Service = "cloudtrail.amazonaws.com"
                        }
                      - Resource  = "arn:aws:sns:us-east-1:<redacted>:lacework-ct-sns-de98b010"
                      - Sid       = "AWSCloudTrailSNSPolicy20150319"
                    },
                ]
                # (1 unchanged element hidden)
            }
        )
        # (2 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```
This doesn’t seem like a workable solution since terraform should be run against an environment on an ongoing basis. Can you provide a solution?
Thanks
**Steps to reproduce**
Outlined in description

**Expected behavior**
After doing the manual attachment it should be ok to run the terraform configuration again without deleting attachment configuration

**Please complete the following information):**
 ```bash
 » tf version                                                                                                            altoira-nonprod
Terraform v1.3.8
on darwin_arm64
+ provider registry.terraform.io/hashicorp/aws v4.54.0
+ provider registry.terraform.io/hashicorp/random v3.4.3
+ provider registry.terraform.io/hashicorp/time v0.9.1
+ provider registry.terraform.io/lacework/lacework v1.4.0
```

**Comment by afiune** [comment]
Hey Duncan, thank you so much for your feedback, let us take a look at this issue. We will come back to you soon.

**Comment by jon-stewart** [comment]
Hi @dmcnaught, this should be fixed with v2.5.1.  Thanks!
